tool/json_to_firebase.py

```diff
@@ -68,6 +71,8 @@ def _usage_and_exit(error_message):
 
     init(CRYPTO_TOKEN_PATH)
 
+    short_id = tool_utils.short_id()
+    log.audit(f"json_to_firebase: JobID {short_id}")
     with open(INPUT_PATH, 'r') as f:
         data_dict = json.load(f)
```

Comment:
Do we want to `log.notify` the top level keys being uploaded to firebase?

Reply:
Good Idea, will adjust, thanks

Reply:
Done